cluster-sizing.md

```diff
@@ -0,0 +1,81 @@
+# Calculating Cluster Dimensions
+
+This is intended as a general guide to determining how many nodes and how much
+storage space per node you require for your workload. Please [contact
+Circonus](/contact.md) if you have questions arising from your specific needs.
+
+## Key Terminology
+
+* `T` is the number of unique metric streams.
+* `N` is the number of nodes participating in the cluster.
+* `W` is the number of times a given measurement is stored across the cluster.
+  * For example, if you have 1 GB of metric data, you must have `W` GB of
+    storage space across the cluster.
+
+The value of `W` determines the number of nodes that can be unavailable before
+metric data become inaccessible. A cluster with `W` write copies can survive
+`W-1` node failures before a partial data outage will occur.
+
+Metric streams are distributed approximately evenly across the nodes in the
+cluster. In other words, each node is responsible for storing approximately
+`(T*W)/N` metric streams. For example, a cluster of 4 nodes with 100K streams
+and `W=2` would store about 50K streams per node.
+
+## Rules of Thumb
+
+* Nodes should be operated at no more than 70% capacity.
+* Favor ZFS striped mirrors over other pool layouts
+* `W` must be >= 2
```

Comment:
There is nothing preventing a W == 1 but it is not recommended.

Reply:
I'd rather not confuse the issue. We have single-node installs which is effectively that, and the common case for clustering is redundancy.